app/javascript/services/pageRoutes.js

```diff
@@ -7,4 +7,9 @@ export const pageRoutes = {
   organizations: () => '/dashboard/organizations',
   configurationProfiles: () => '/dashboard/configuration-profiles',
   configurationProfile: (id) => `/dashboard/configuration-profiles/${id}`,
+  // mappings
+  mappingsList: (cp, abstractClass = null) =>
+    `/mappings-list${cp ? `?cp=${cp}` : ''}${
```

Comment:
How about using `query-string` here instead of composing the param  manually?

Reply:
Good idea, will address at next PRs.